### PR TITLE
Support interactive cli testing via input file

### PR DESCRIPTION
**Pull request by mdeuser**
Update test framework to allow stdin input from a file

### CODE REVIEW

**Comment by mdeuser** [comment]
PG3/672

**Comment by mdeuser** [comment]
@csantanapr - ready for review

/cc @dubeejw @rabbah

File: tests/src/test/scala/common/Wsk.scala [hunk 1]
```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```
Comment (markusthoemmes) [inline tests/src/test/scala/common/Wsk.scala]: That should be an `Option[File]` with a default value of `None`.

Then to use it against the Java API you can use `stdinFile.getOrElse(null)`. At least keeps the `null` out of the Scala API.

Comment (mdeuser) [inline tests/src/test/scala/common/Wsk.scala]: With all other parameters are not `Option` parameters, would introducing an `Option` parameter be confusing?

Comment (mdeuser) [inline tests/src/test/scala/common/Wsk.scala]: perhaps it's just me, but method signatures with a mix of `Option` parameters and named default parameters is a bit confusing.
https://stackoverflow.com/questions/4199393/are-options-and-named-default-arguments-like-oil-and-water-in-a-scala-api

Comment (rabbah) [inline tests/src/test/scala/common/Wsk.scala]: It avoids introducing nulls and hence NPEs. Here's it's fairly localized and passed on to the Java client, so it's not a big deal. FWIW we have explored introducing a scala linter which will flag uses of null. So personally I'd change it to an Option type also.

Comment (mdeuser) [inline tests/src/test/scala/common/Wsk.scala]: another scala best practice recommendation to remove `null` from all scala code... removing `null` per @markusthoemmes suggestion :+1: 
http://alvinalexander.com/scala/scala-null-values-option-uninitialized-variables

**Comment by mdeuser** [comment]
rebased and squashed.  PG3/674

**Pull request merged**